docs/assets/theme-custom.css

```diff
@@ -23,4 +23,46 @@
     --pagination-title-font-size: var(--font-size-m);
     --pagination-chevron-height: 2em;
     --pagination-chevron-stroke-width: 1.5px; 
+}
+
+@font-face {
+    font-family: "72";
+    src: url("https://cdn.jsdelivr.net/npm/@sap-theming/theming-base-content/content/Base/baseLib/baseTheme/fonts/72-Regular-full.woff") format("woff");
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: "72";
+    src: url("https://cdn.jsdelivr.net/npm/@sap-theming/theming-base-content/content/Base/baseLib/baseTheme/fonts/72-Bold-full.woff") format("woff");
+    font-weight: 700;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: "72";
+    src: url("https://cdn.jsdelivr.net/npm/@sap-theming/theming-base-content/content/Base/baseLib/baseTheme/fonts/72-Light-full.woff") format("woff");
+    font-weight: 300;
+    font-style: normal;
+}
+
+@font-face {
```

Comment:
I don't think you need to load **SAP-icons** nor **BusinessSuiteInAppSymbols**, nor **SAP-icons-TNT**.
If you are not using them you can remove them